lib/src/Base/Algo/ClassifierImplementation.cxx

```diff
@@ -58,9 +58,15 @@ ClassifierImplementation * ClassifierImplementation::clone() const
 }
 
 
+UnsignedInteger ClassifierImplementation::getClassNumber() const
+{
+  throw NotYetImplementedException(HERE) << "In ClassifierImplementation::getClassNumber() const";
+}
+
 UnsignedInteger ClassifierImplementation::getNumberOfClasses() const
 {
-  throw NotYetImplementedException(HERE) << "In ClassifierImplementation::getNumberOfClasses() const";
+  LOGWARN(OSS() << "getNumberOfClasses is deprecated, use getClassNumber");
```

Comment:
The problem is that `getClassNumber` is an ambiguous name: it could mean "the number of this class", i.e. an ID for a specific class. I prefer the old name.

Reply:
ok